### PR TITLE
Add Warning indicator.

### LINKED ISSUE #9125: Add indicator on nodes with warnings
- Split out of #8678 as the corresponding PR was merged without support for this specific feature.

The UI should (?) match with the existing orange border present on GUI1.

### PR DESCRIPTION
**Pull request by somebody1234**
### Pull Request Description
- Close #9125
  - Display the new warning icon to the left of the node, when the node is not being hovered over.

### Important Notes
None

### Screenshots
Not hovered:
image

Hovered:
- the warning icon MUST be hidden when the node is hovered, otherwise it overlaps with the node name.
  - this is fine, as the full warning is shown on hover anyway.

image

Using new warning icon for warning messages:
image

Using new error icon for error messages:
image

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] ~~The documentation has been updated, if necessary.~~
- [x] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- All code has been tested:
  - [x] ~~Unit tests have been written where possible.~~
  - [x] ~~If GUI codebase was changed, the GUI was tested when built using `./run ide build`.~~

### CODE REVIEW

**Comment by somebody1234** [comment]
the collapsed state is slightly wider than it is tall, which i'm not 100% satisfied with...

**Comment by somebody1234** [comment]
nvm fixed

File: app/gui2/src/components/GraphEditor/GraphNode.vue [hunk 1]
```diff
@@ -114,7 +114,7 @@ const warning = computed(() => {
   const info = projectStore.computedValueRegistry.db.get(externalId)
   const warning = info?.payload.type === 'Value' ? info.payload.warnings?.value : undefined
   if (!warning) return
-  return '⚠ Warning: ' + warning!
+  return '⚠\xa0\xa0Warning: ' + warning!
```
Comment (kazcw) [inline app/gui2/src/components/GraphEditor/GraphNode.vue]: Let's use a named constant for `\xa0`, some of us (like me) are HTML noobs who can't recognize an *nbsp* by its hex code.

**Review by kazcw — APPROVED** [review]

File: app/gui2/src/components/GraphEditor/GraphNode.vue (hunk 1 above)
Comment (somebody1234) [reply app/gui2/src/components/GraphEditor/GraphNode.vue]: good catch, thanks!

**Comment by somebody1234** [comment]
nvm. we have an actual icon (plus a more proper spec) for this task now

**Comment by somebody1234** [comment]
"re-requesting review" from kaz mainly to remove the approval - this is because this PR has been completely reimplemented because the new icons have now been merged in.

**Comment by somebody1234** [comment]
@jdunkerley so i just noticed you mentioned it should appear to the left of the menu, but i'm not sure it's a good idea (see screenshots). is this fine? or perhaps do you have a better solution? (should the name be shifted left too?)

**Comment by somebody1234** [comment]
also - should the new icon also be used as the icon for the warnings viz? or should the warnings viz be removed completely?

i suppose it should stay until we have UI for the full list of warnings. (unless the warnings viz *is* the full UI, but i was under the impression that it was just a quick workaround)

**Review by farmaazon — APPROVED** [review]

**Pull request merged**